src/App.js

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect, useState } from 'react';
+import {
+  BrowserRouter,
+  Routes,
+  Route,
+} from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { getCryptos } from './redux/cryptos/cryptos';
+import Nav from './components/Nav';
+import CryptoList from './components/CryptoList';
+import CryptoDetails from './components/CryptoDetails';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import './App.css';
+
+function App() {
```

Comment:
- [ ] OPTIONAL
It is a nice idea to prefer _arrow functions_ over normal functions, so as to keep your _function_ syntax simpler and cleaner.